include/aunit/framework/aunit-reporter.ads

```diff
@@ -29,12 +29,17 @@
 --                                                                          --
 ------------------------------------------------------------------------------
 
+with Ada.Text_IO;
```

Comment:
Ada.Text_IO is not available on most restricted runtimes like zfp for example. If you wich to take advantage of it you need to introduce different versions of affected packages for different runtime presets like is currently done for calendar, exceptions and memory management.